lib/region/europe/countryCodes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file List of country codes legally considered part of the EU region.
+ */
+
+var countryCodes = {
```

Comment:
I'm pretty sure I get it, but I'd still like to ask--what's the reasoning behind using this sort of data structure instead of a list?